pages/tapping/index.js

```diff
@@ -29,11 +29,11 @@ const StyledTappingGameField = styled(StyledGameField)`
 `;
 
 const TappingCircle = styled.button`
-  background-image: ${({ isActive, isWrongActive }) =>
+  background-image: ${({ isActive, petImage, isWrongActive }) =>
     isWrongActive
-      ? `url("/images/ghost.png")`
+      ? `url("/assets/images/ghost_front.png")`
       : isActive
```

Comment:
props passed to styled components should be prefixed with "$". You should change it in this component and also adjust the code where you call this component.